src/js/components/encodings/GroupSubMenu.jsx

```diff
@@ -23,11 +27,27 @@ function mapStateToProps(reduxState, ownProps) {
     group: getIn(reduxState, 'marks.' + ownProps.id)
   };
 }
-
 function mapDispatchToProps(dispatch, ownProps) {
   return {
-    select: function(id) {
-      dispatch(selectMark(id));
+    select: function(id, encType) {
+      // either name selectMark selectEncoding or create selectGroup action
+      switch (encType) {
+        case 'mark':
+          dispatch(selectMark(id));
+          break;
+        case 'guide':
+          dispatch(selectGuide(id, ownProps.id));
```

Comment:
If we remove `groupId` from `selectGuide`, make sure to remove the argument here as well.